agreement/demux.go

```diff
@@ -323,6 +375,17 @@ func (d *demux) next(s *Service, deadline time.Duration, fastDeadline time.Durat
 		d.UpdateEventsQueue(eventQueueTokenized[protocol.ProposalPayloadTag], 0)
 		d.monitor.inc(demuxCoserviceType)
 		d.monitor.dec(tokenizerCoserviceType)
+	case m, open := <-d.txnsyncProposals:
+		if !open {
+			logging.Base().Warnf("channel was closed")
```

Comment:
`logging.Base().Warnf` -> `s.log.with(logEvent).Warnf`

Reply:
Thanks, just letting you know that I have a bunch of logging messages that are still in the branch mostly just for testing